perf/samples.js

```diff
@@ -2,9 +2,7 @@
 
   function Samples(count) {
     this.startTime = 0;
-
-    this.data = new Array(count).fill(0);
-    this.data.length = 0;
+    this.data = [];
```

Comment:
Sure, but I'm not aware of any implementation of dynamically growing arrays that ever shrinks. In any case, better to isolate GCs due to reallocation from the tests in browsers that don't deallocate when truncating (which I am guessing is all of them).